tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -291,6 +291,104 @@ StorageManagerCanonical::load_array_schemas_and_fragment_metadata(
       Status::Ok(), array_schema_latest, array_schemas_all, fragment_metadata};
 }
 
+std::tuple<
+    Status,
+    optional<shared_ptr<ArraySchema>>,
+    optional<std::unordered_map<std::string, shared_ptr<ArraySchema>>>,
+    optional<std::vector<shared_ptr<FragmentMetadata>>>>
+StorageManager::load_array_schemas_and_all_fragment_metadata(
+    const ArrayDirectory& array_dir,
+    MemoryTracker* memory_tracker,
+    const EncryptionKey& enc_key) {
+  auto timer_se =
+      stats_->start_timer("sm_load_array_schemas_and_all_fragment_metadata");
+
+  // Load array schemas
+  auto&& [st_schemas, array_schema_latest, array_schemas_all] =
+      load_array_schemas(array_dir, enc_key);
+  RETURN_NOT_OK_TUPLE(st_schemas, std::nullopt, std::nullopt, std::nullopt);
+
+  auto xfiltered_fragment_uris = array_dir.filtered_fragment_uris(
```

Comment:
xfiltered_fragment_uris -> filtered_fragment_uris

Reply:
not needed, removed